app/src/models/github-repository.ts

```diff
@@ -58,6 +58,11 @@ export class GitHubRepository {
   public get fork(): boolean {
     return !!this.parent
   }
+
+  /** Is the repository a GHE repository */
+  public get isGHE(): boolean {
```

Comment:
So `ghe.io` is our own internal GitHub Enterprise Server (GHES) installation. For a customer the endpoint would typically be somethine like `https://github.company.com/api/v3`. So what we typically do everywhere we need to differentiate GitHub.com from non GitHub.com endpoints (which up until the advent of GHAE was only GHES) is to check if the endpoint matches .com's API endpoint and if it doesn't we'll know that it's either GHAE or GHES.